src/OmniSharp/Workers/Intellisense/SnippetGenerator.cs

```diff
@@ -75,9 +78,9 @@ private void RenderMethodSymbol(IMethodSymbol methodSymbol)
                 }
                 _sb.Append(">");
             }
-            
+
             RenderParameters(methodSymbol);
-            if (methodSymbol.ReturnsVoid)
+            if (methodSymbol.ReturnsVoid && IncludeMarkers)
```

Comment:
https://github.com/OmniSharp/omnisharp-roslyn/blob/master/tests/OmniSharp.Tests/SnippetFacts.cs#L319 thats the `Snippet` not the `CompletionText`